packages/cli/internal/pkg/aws/cfn/stack_filters.go

```diff
@@ -24,3 +24,20 @@ var ActiveStacksFilter = []types.StackStatus{
 	types.StackStatusImportRollbackFailed,
 	types.StackStatusImportRollbackComplete,
 }
+
+var QueryableStacksMap = map[types.StackStatus]bool{
+	types.StackStatusCreateComplete:                          true,
+	types.StackStatusUpdateInProgress:                        true,
+	types.StackStatusUpdateCompleteCleanupInProgress:         true,
+	types.StackStatusUpdateComplete:                          true,
+	types.StackStatusUpdateRollbackInProgress:                true,
+	types.StackStatusUpdateRollbackFailed:                    true,
+	types.StackStatusUpdateRollbackCompleteCleanupInProgress: true,
+	types.StackStatusUpdateRollbackComplete:                  true,
+	types.StackStatusReviewInProgress:                        true,
+	types.StackStatusImportInProgress:                        true,
+	types.StackStatusImportComplete:                          true,
+	types.StackStatusImportRollbackInProgress:                true,
+	types.StackStatusImportRollbackFailed:                    true,
+	types.StackStatusImportRollbackComplete:                  true,
+}
```

Comment:
Could this be merged with the above list?

```go
type StackStatusInfo struct {
	active    bool
	queryable bool
}

var StackStatusInfos = map[string]StackStatusInfo{
	"ACTIVE": {active: true, queryable: true},
}
```

Reply:
I actually think that the list above is incorrect (deleting/deploying stacks are considered "active" as well as stacks that failed to delete which could have any number of resources inside of them) but wanted to have a discussion about it as a group.